.lintrunner.toml

```diff
@@ -223,6 +223,7 @@ include_patterns = [
     'c10/**/*.cpp',
     'c10/**/*.h',
     'torch/*.h',
+    'torch/stable/*.h',
```

Comment:
IMO all C/C++ headers should be in `torch/csrc` folder, also `stable` is, IMO, confusing name. How about `capi`?

Reply:
The C API is the shim.h in aoti/c/, this directory is meant to host the C++ header only files that will only call into the shim.h API or be self contained.